composer.json

```
@@ -5,23 +5,23 @@
     "type": "symfony-bundle",
     "license": "LGPL-2.1",
     "require": {
-        "php": ">=7.2.0",
+        "php": ">=8.0.0",
         "ext-json": "*",
         "oat-sa/lib-lti1p3-core": "^6.0",
-        "psr/log": "^1 || ^2 || ^3",
-        "symfony/framework-bundle": "^4.4 || ^5.1",
+        "psr/log": "^2 || ^3",
```

While it does make sense to drop support for `psr/log`, now that PHP 7 support is getting dropped, and this is a breaking change anyway, I still would like to leave version 1 as an option here.
I will help in organizing migration in other repositories of ours.

Thanks!
```suggestion
        "psr/log": "^1 || ^2 || ^3",
```